packages/@aws-cdk/aws-appmesh/README.md

```diff
@@ -590,3 +590,45 @@ appmesh.Mesh.fromMeshArn(stack, 'imported-mesh', arn);
 ```ts
 appmesh.Mesh.fromMeshName(stack, 'imported-mesh', 'abc');
 ```
+
+## IAM Grants
+
+Each mesh resource includes 2 methods to generate IAM policy documents. `grantRead` grants the user read access to the 
+resource all of it's subresources. And similarly `grantWrite` grants the user write access to the resource to all 
+subresources. Virtual Node and Virtual Gateway implement a separate `grantStreamAggregatedResources` method for 
+identities that are running Envoy and need to access generated config.
+
+```ts
+const gateway = new appmesh.VirtualGateway(stack, 'testGateway', { mesh: mesh });
+const gatewayAdminUser = new iam.User(stack, 'gatewayAdminUser');
+const envoyUser = new iam.User(stack, 'envoyUser');
+
+/**
+ * This will grant the user the following actions:
+ * - appmesh:CreateVirtualGateway
+ * - appmesh:UpdateVirtualGateway
+ * - appmesh:DeleteVirtualGateway
+ * - appmesh:CreateGatewayRoute
+ * - appmesh:UpdateGatewayRoute
+ * - appmesh:DeleteGatewayRoute
+ * - appmesh:TagResource
+ * - appmesh:UntagResource
```

Comment:
I feel like we're just asking for this to become stale ;p. Maybe we don't have to include the entire list here...?